File: typescript/src/main/vsdk/toolkit/common/commandLineOptions/OptionGroup.ts
import { OptionBase } from "./OptionBase";
import { OptionGroupT } from "./OptionGroupT";

export class OptionGroup extends OptionGroupT<OptionBase> {
  public constructor();
  public constructor(groupName: string, groupOptions: OptionBase[], groupCount: number);
  public constructor(groupName?: string, groupOptions: OptionBase[] | null = null, groupCount = 0) {
    if (groupName === undefined) {
      super();
      return;
    }
    super(groupName, groupOptions ?? [], groupCount);
  }
}

File: java/base/src/main/java/vsdk/toolkit/common/commandLineOptions/OptionGroup.java
package vsdk.toolkit.common.commandLineOptions;

public class OptionGroup extends OptionGroupT<OptionBase> {
    public OptionGroup() {
        super();
    }

    public OptionGroup(String groupName, OptionBase[] groupOptions, int groupCount) {
        super(groupName, groupOptions, groupCount);
    }
}
